duckduckgo/Android
# Update autofill to 3.2.3

Task/Issue URL: https://app.asana.com/0/1125189844152671/1200635726526715
Tech Design URL: 
CC: 

**Description**:
This PR updates autofill to 3.2.3. 

You **might** need to run this command to update the submodule with the correct commit, but ignore this step if the commit hash is correct.
`git submodule update submodules/autofill`

**Steps to test this PR**:
1. Checkout branch
1. Run `git submodule`
1. Confirm autofill commit hash is`53b3a7941e4dd2e362dc94992e6bdd748fa3c8dd `

**Autofill still works**
1. Sign in with your duck address
1. After signing all the email functionalities should be there and work correctly:
1. Create Duck Address in the options menu should copy a private address in the clipboard.
1. Go to `github.com`, autofill should appear when clicking on Dax
1. Go to the Asana task and run the test cases from there.
---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
